plugins/chat/lib/chat/channel_fetcher.rb

```diff
@@ -251,29 +247,13 @@ def self.tracking_state(channel_ids, guardian, include_threads: false)
     end
 
     def self.find_with_access_check(channel_id_or_name, guardian)
```

Comment:
Probably makes sense to change this to `channel_id_or_slug` now that it's not using name